snowtrick/public/js/detail.js

```diff
@@ -0,0 +1,83 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+        // add event on trigger button
+    function activeTrashButton() {
+        const trash_btn = document.querySelectorAll(".trash_btn");
+        trash_btn.forEach((trash) => {
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/trousse/OC_Projet6/pullRequest?prid=10057695)
```suggestion
    trash_btn.forEach((trash) => {
```